crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -487,7 +487,7 @@ where
             // we need to check if the parent block is the last POW block, if so then the payload is
             // the first POS. The engine API spec mandates a zero hash to be returned: <https://github.com/ethereum/execution-apis/blob/6709c2a795b707202e93c4f2867fa0bf2640a84f/src/engine/paris.md#engine_newpayloadv1>
             if parent_header.difficulty != U256::ZERO {
-                return Some(B256::ZERO)
+                return Some(B256::ZERO);
```

Comment:
```suggestion
                return Some(B256::ZERO)
```

Reply:
Addressed in https://github.com/anton-rs/op-reth/pull/145